internal/nonkube/bundle_common.go

```diff
@@ -0,0 +1,21 @@
+package nonkube
```

Comment:
Suggest bundle as its own package and putting files into the package directory as other nonkube packages might show up over time